### PR TITLE
Fix pin versions nyc version for testing on node 8 & 9

### PR DESCRIPTION
**Pull request by carpasse**
Same fix as jshttp/http-errors#109

Failing build https://github.com/carpasse/statuses/actions/runs/8523124039
Build working with the fix https://github.com/carpasse/statuses/actions/runs/8523443522

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -73,32 +75,37 @@ jobs:
...
         - name: Node.js 15.x
-          node-version: "15.5"
+          node-version: "15.14"
 
         - name: Node.js 16.x
-          node-version: "16.2"
+          node-version: "16.13"
+
+        - name: Node.js 17.x
+          node-version: "17.2"
```
Comment (UlisesGascon) [inline .github/workflows/ci.yml:108]: Can we move this to a separate PR? Or include up to 21 in this one? I will suggest to keep the PR the most atomic possible and avoid add new Node.js versions in this fix.

Comment (carpasse) [reply .github/workflows/ci.yml:108]: I will move it to a separate PR

Comment (carpasse) [reply .github/workflows/ci.yml]: removed

Comment (carpasse) [reply .github/workflows/ci.yml]: PR adding new versions https://github.com/jshttp/statuses/pull/32

**Comment by UlisesGascon** [comment]
I think that this PR content was covered by https://github.com/jshttp/statuses/pull/32, so I will close this PR. @carpasse feel free to re open the PR if needed :-)

**Pull request closed without merging**